### PR TITLE
wacd test

### PR DESCRIPTION
**Pull request by simcha-waintrob**

### CODE REVIEW

File: server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            expect(metaDataResource.MetaData).to.be.an('Array').that.is.not.empty
+            // MetaData is an array with only one element
+            expect(metaDataResource.MetaData.length).to.equal(1)
+            const mapDataMetaData = metaDataResource.MetaData[0]
+            expect(mapDataMetaData).to.have.property('ObjectID.ID').that.is.a('Number').and.is.not.undefined
+            expect(mapDataMetaData).to.have.property('Hidden').that.is.a('Boolean').and.is.not.undefined
+            expect(mapDataMetaData).to.have.property('Key1Type').that.is.a('Number').and.is.not.undefined
+            expect(mapDataMetaData).to.have.property('Key2Type').that.is.a('Number').and.is.not.undefined
+            expect(mapDataMetaData).to.have.property('ExternalID').that.is.a('String').and.is.not.undefined
+            expect(mapDataMetaData).to.have.property('Configuration').that.is.a('String').and.is.not.undefined
```
Comment (liorSela) [inline server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts:32]: shouldn't you also verify that configuration is a valid json?

File: server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                expect(mapData).to.have.property('Hidden').that.is.a('Boolean').and.is.not.undefined
+                expect(mapData).to.have.property('MapDataMetaData.ObjectID.ID').that.is.a('Number').and.is.not.undefined
+                expect(mapData).to.have.property('ObjectID.ID').that.is.a('Number').and.is.not.undefined
+                // test that values is a valid JSON
+                expect(GlobalSyncService.isValidJSON(mapData.Values)).to.be.true
+            }
+        }
+        return Promise.resolve()
+    }
+
```
Comment (liorSela) [inline server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts:49]: where do you check that the schema you created exists in the wacd data?

Comment (simcha-waintrob) [reply server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts:49]: you right, i'lll call the super to make sure that schema is exist

Comment (liorSela) [reply server-side/tests/api_tests/services/sync/test-commands/wacd-command.ts:53]: it wouldn't work, the schema exists is in a different structure and the wacd data don't have the properties that we check in schema exists,
Also, the super test has failed but the test itself still passed so we should avoid calling super in the test function

**Review by liorSela — APPROVED** [review]

**Pull request merged**